index.html

```diff
@@ -1,21 +1,159 @@
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/common.css" />
+    <link rel="stylesheet" href="css/main.css" />
     <title>Responsive grid project</title>
-</head>
-
-<body>
+  </head>
 
+  <body>
     <!-- Add your markup here -->
+    <header>
+      <nav>
+        <ul>
+          <li>Shop</li>
+          <li>About</li>
+          <li>FAQ</li>
+          <li>Contact</li>
+        </ul>
+      </nav>
+      <p>Call Us Now! 123-456-7890</p>
+    </header>
+    <main>
+      <section class="hero">
+        <div class="hero-text">
+          <h1>PRICKLES & CO</h1>
+          <p>BRING NATURE INDOORS</p>
+          <button>Shop Now</button>
+        </div>
+      </section>
 
+      <section class="collections">
+        <div class="collection">
+          <p>CACTI</p>
+          <button>Shop Collection</button>
+          <img src="" />
+        </div>
 
-    <!-- Please leave this in to credit the creators of this design -->
-    <p>This website design was created by Wix.com, and is used here for strictly educational purposes.</p>
-</body>
+        <div class="collection">
+          <p>PLANTS</p>
+          <button>Shop Collection</button>
+          <img src="" />
+        </div>
+
+        <div class="collection">
+          <p>SUCCULENTS</p>
+          <button>Shop Collection</button>
+          <img src="" />
+        </div>
+      </section>
+
+      <section class="about-us">
+        <div class="about-us-row">
+          <div class="about-us-text">
+            <p>ABOUT US</p>
+            <p>
+              I'm a paragraph. Click here to add your own text and edit me. It’s
+              easy. Just click “Edit Text” or double click me to add your own
+              content and make changes to the font. Feel free to drag and drop
+              me anywhere you like on your page. I’m a great place for you to
+              tell a story and let your users know a little more about you.
+            </p>
+            <p>
+              This is a great space to write long text about your company and
+              your services. You can use this space to go into a little more
+              detail about your company. Talk about your team and what services
+              you provide.
+            </p>
+          </div>
+          <img src="" />
+          <img src="" />
+        </div>
+        <div class="buy-online-row">
+          <div class="buy-online-plant-image">
+            <img src="" />
+          </div>
+          <div class="buy-online-info">
+            <p>BUY ONLINE NOW & GET 10% OFF!</p>
+            <button>Shop Now</button>
+          </div>
+        </div>
+      </section>
+
+      <section class="special-requests">
+        <div class="requests-box">
+          <p>FOR SPECIAL REQUESTS & ORDERS</p>
+          <div class="requests-form">
+            <form></form>
+          </div>
+        </div>
+      </section>
+      <footer>
+      <section class="contact-details">
```

Comment:
Do you need this `section` here? It seems to wrap exactly the same content as the footer, so you can just apply your class to the footer directly and avoid the extra markup.